test/pytest/test_utils.py

```diff
@@ -12,6 +12,7 @@
 from queue import Queue
 from subprocess import PIPE, STDOUT, Popen
 
+import orjson
```

Comment:
nit: Do we need this? Can't we achieve the same with `json`?

Reply:
According to the [description](https://github.com/ijl/orjson?tab=readme-ov-file#orjson),  it is fastest json lib. I'm thinking if we need apply orjson in our handler for json input.